priv/blocks/to_json.json

```diff
@@ -12,6 +12,17 @@
 
     "additionalProperties": false,
 
-    "properties": {}
+    "properties": {
+      "pretty": {
+        "type": "boolean",
+        "description": "Serialize the output to pretty format that is easier to read for humans.",
+	"default": "false"
+      },
+      "template": {
+        "type": "object",
+        "description": "Apply given JSONTemplate.",
+        "default": "null"
```

Comment:
It's not required so no need to provide a default (moreover that's the `"null"` string, not `null`)